annotated-projects/PXD001819/PXD001819.sdrf.tsv

```diff
@@ -1,28 +1,28 @@
+Sample 4	Saccharomyces cerevisiae	not available	not available	not applicable	2 mg	CT=mixture;QY=2500 amol;CN=UPS1;CV=Standards Research Group	1	lysate	run 11	proteomic profiling by mass spectrometry	AC=MS:1002038;NT=label free sample	AC=MS:1001742;NT=LTQ Orbitrap Velos	5 ppm	0.8 Da	NT=trypsin/P;AC=MS:1001313	NT=Carbamidomethyl;TA=C;MT=fixed;AC=UNIMOD:4	NT=Oxidation;MT=variable;TA=M;AC=UNIMOD:35	NT=Acetyl;AC=UNIMOD:67;PP=Protein N-term;MT=variable	2	1	ftp://ftp.ebi.ac.uk/pride-archive/2015/12/PXD001819/UPS1_2500amol_R2.raw	UPS1_2500amol_R2.raw	CT=mixture;QY=2500 amol;CN=UPS1;CV=Standards Research Group
+Sample 4	Saccharomyces cerevisiae	not available	not available	not applicable	2 mg	CT=mixture;QY=2500 amol;CN=UPS1;CV=Standards Research Group	1	lysate	run 12	proteomic profiling by mass spectrometry	AC=MS:1002038;NT=label free sample	AC=MS:1001742;NT=LTQ Orbitrap Velos	5 ppm	0.8 Da	NT=trypsin/P;AC=MS:1001313	NT=Carbamidomethyl;TA=C;MT=fixed;AC=UNIMOD:4	NT=Oxidation;MT=variable;TA=M;AC=UNIMOD:35	NT=Acetyl;AC=UNIMOD:67;PP=Protein N-term;MT=variable	3	1	ftp://ftp.ebi.ac.uk/pride-archive/2015/12/PXD001819/UPS1_2500amol_R3.raw	UPS1_2500amol_R3.raw	CT=mixture;QY=2500 amol;CN=UPS1;CV=Standards Research Group
+Sample 5	Saccharomyces cerevisiae	not available	not available	not applicable	2 mg	CT=mixture;QY=250 amol;CN=UPS1;CV=Standards Research Group	1	lysate	run 13	proteomic profiling by mass spectrometry	AC=MS:1002038;NT=label free sample	AC=MS:1001742;NT=LTQ Orbitrap Velos	5 ppm	0.8 Da	NT=trypsin/P;AC=MS:1001313	NT=Carbamidomethyl;TA=C;MT=fixed;AC=UNIMOD:4	NT=Oxidation;MT=variable;TA=M;AC=UNIMOD:35	NT=Acetyl;AC=UNIMOD:67;PP=Protein N-term;MT=variable	1	1	ftp://ftp.ebi.ac.uk/pride-archive/2015/12/PXD001819/UPS1_250amol_R1.raw	UPS1_250amol_R1.raw	CT=mixture;QY=250 amol;CN=UPS1;CV=Standards Research Group
+Sample 5	Saccharomyces cerevisiae	not available	not available	not applicable	2 mg	CT=mixture;QY=250 amol;CN=UPS1;CV=Standards Research Group	1	lysate	run 14	proteomic profiling by mass spectrometry	AC=MS:1002038;NT=label free sample	AC=MS:1001742;NT=LTQ Orbitrap Velos	5 ppm	0.8 Da	NT=trypsin/P;AC=MS:1001313	NT=Carbamidomethyl;TA=C;MT=fixed;AC=UNIMOD:4	NT=Oxidation;MT=variable;TA=M;AC=UNIMOD:35	NT=Acetyl;AC=UNIMOD:67;PP=Protein N-term;MT=variable	2	1	ftp://ftp.ebi.ac.uk/pride-archive/2015/12/PXD001819/UPS1_250amol_R2.raw	UPS1_250amol_R2.raw	CT=mixture;QY=250 amol;CN=UPS1;CV=Standards Research Group
```

Comment:
@julianu can you also follow the standard that we use before PXD{}-Sample-{index}